controller/persistence/session_store_test.go

```diff
@@ -52,16 +52,16 @@ func (ctx *TestContext) testCreateInvalidSessions(_ *testing.T) {
 	service := ctx.RequireNewService("test-service")
 
 	session := NewSession("", service.Id)
-	err := ctx.Create(session)
-	ctx.EqualError(err, "fk constraint on sessions.apiSession does not allow null or empty values")
+	//	err := ctx.Create(session)
```

Comment:
dead code in a test? on purpose ? if actually dead let's just remove?

Reply:
Fixed, uncommented.